server.js

```diff
@@ -18,7 +36,137 @@ app.use(express.json());
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send("Hello! This is the backend-part of a previous project called Happy Thoughts.");
+});
+
+app.get('/thoughts', async (req, res) => { 
+//V1 Mongoose   
+  const { page } = req.query;
+  try {
+    const thoughts = await Thought.find({})
+    .sort({createdAt: -1})
+    .skip((page -1) * 20).limit(20);
```

Comment:
This code block is a proof of that you've now learned sort and pagination - great work!